corehq/apps/hqwebapp/templates/hqwebapp/crispy/field/field_with_text.html

```diff
@@ -9,7 +9,6 @@
         </label>
         <div class="controls" style="padding-top: 5px;">
             <input type="hidden"
-                   name="{{ field.html_name }}"
```

Comment:
I'm also concerned that this will result in the value not being submitted to the server (since the 'name' attr is what's used to identify the value)